web/src/client/index_test.ts

```diff
@@ -877,7 +877,7 @@ describe("POST /installer/validate", () => {
       } catch (error) {
         err = error;
       }
-      expect(err).to.have.property("message", "Rook versions >= 1.4.0 require blockStorageEnabled to be set to true");
+      expect(err).to.be.undefined;
```

Comment:
honestly a lot of these tests can probably be removed entirely now that we're getting rid of the check